packages/sdk-redux/package.json

```diff
@@ -34,7 +34,8 @@
     },
     "dependencies": {
         "ethers": "^5.5.1",
-        "lodash": "^4.17.21"
+        "lodash": "^4.17.21",
+        "graphql-request": "^3.7.0"
```

Comment:
if we have time, we can revisit our monorepo hoisting strategy to resolve this type of issue in the future https://classic.yarnpkg.com/blog/2018/02/15/nohoist/